WORKSPACE

```diff
@@ -67,7 +67,7 @@ http_archive(
 # b) by commenting out the http_archive above and uncommenting the following:
 # local_repository(
 #    name = "xla",
-#    path = "/path/to/openxla",
+#    path = "/home/lsiyuan/work/xla",
```

Comment:
Is this intended?

Reply:
Not intended, for local testing purpose. I'll recover it.